src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -33431,6 +33759,12 @@ void M4([NotNull]out int? x)
     {
         return null; // 3
     }
+
+    void M7([NotNull]out int? x)
+    {
+        x = null;
+        return; // 4
+    }
```

Comment:
Please add a few cases for constructors where the only code in the method is a `:this` call to another constructor and verify that we validate not-null correctly in those cases.

Reply:
I wasn't sure exactly what this meant--do you want tests where constructor parameters have `[NotNull]` and `[NotNullIfNotNull]`?